src/shared/components/Resources/ResourceActions.tsx

```diff
@@ -121,6 +121,7 @@ export interface ResourceActionsProps {
   goToElasticSearchView: (resource: Resource) => void;
   deprecateResource: (resource: Resource) => void;
   downloadFile: (resource: Resource) => void;
+  veryDangerousToDeprecate: (resource: Resource) => void;
```

Comment:
What's the difference between `deprecateResource` and `veryDangerousToDeprecate` ?

Reply:
one is the normal message, "are you sure?", and the other is "this will break stuff!"

Reply:
I mean... we're not going to add a new prop everytime we need a different message right? can we not do something like:
```suggestion
  deprecateResource: (resource: Resource, message?: string) => void;
```

Reply:
You're right, I fixed the actions to be a little more generic, now you can create a button that uses another action and you dont have to add a new prop type every time